.github/workflows/python-package-develop.yml

```diff
@@ -79,6 +76,9 @@ jobs:
   test-container:
     runs-on: ubuntu-latest    
     needs: ['build']
+    permissions:
```

Comment:
Val this should be moved from the previous lines,

Need to add the permission I mentioned on slack here as an addition